spacewalk/setup/bin/spacewalk-setup

```diff
@@ -1163,8 +1163,24 @@ sub setup_services {
   return 1;
 }
 
+sub is_suse_like {
+  my $line;
+  if(! -e '/etc/os-release') {
+    return undef;
+  }
+  open(OS_RELEASE,"/etc/os-release") || die ("Could not open /etc/os-release\n");
+  while( $line = <OS_RELEASE> ) {
+    if ($line =~ m/ID_LIKE=".*suse.*"/) {
```

Comment:
Why not just `ID_LIKE="suse.*"/`? It seems from your examples that `ID_LIKE` always starts with `suse*`

Reply:
Just in case, as I don't trust it will be always like that.